packages/types/src/interfaces/injects.ts

```diff
@@ -24,6 +25,7 @@ const _types = {
   other: {
     types: {
       ExtrinsicSignatureV4: CENNZnetExtrinsicSignatureV1,
+      VecDeque: VecAny,
```

Comment:
https://github.com/polkadot-js/api/blob/ffaea83e3eff25787fd1a1d0660d9d0bd7bdced6/packages/types/README.md

[[VecAny]] | This manages codec arrays, assuming that the inputs are already of type Codec
-- | --
https://github.com/polkadot-js/api/blob/33978d5007becbca5884862968eb778a866b6a94/packages/types/src/codec/VecAny.ts#L15